config/owasp/suppressions.xml

```diff
@@ -6,6 +6,11 @@
     <cve>CVE-2023-4586</cve>
   </suppress>
 
+  <suppress until="2024-01-14">
```

Comment:
i think Jack has asked us to stop suppressing these warnings, and instead force it to the latest version to fix it, e.g.:-
https://github.com/hmcts/darts-api/blob/master/build.gradle#L417

Reply:
Its the latest version thats bringing it in, why the suppression is temporary